.circleci/docker/common/install_base.sh

```diff
@@ -77,6 +77,7 @@ install_centos() {
     glog-devel \
     hiredis-devel \
     libstdc++-devel \
+    libsndfile-devel \
```

Comment:
can you also add it to /docker/caffe2/jenkins/common/install_base.sh ?

Reply:
I didn't realize the dockerfiles under ./docker were still used?  You'll notice sndfile is missing there for both ubuntu and centos.

Reply:
If you look at the commit history under https://github.com/pytorch/pytorch/commits/master/docker, these files really haven't been touched much except for by me, and at that time we hadn't yet migrated to the dockerfiles under .circleci.  So my concern is that this ./docker directory is quite stale and should have an audit to see if it is still useful and necessary.

Reply:
hmm. interesting. AFAIK we have some new ways to deal with docker images. let me try to see if this is still a needed dir.